src/core/decrypt/init_media_keys.ts

```diff
@@ -47,14 +45,7 @@ export default async function initMediaKeys(
 
   if (shouldDisableOldMediaKeys) {
     log.debug("DRM: Disabling old MediaKeys");
-    // TODO should we be awaiting always?
-    // Should be tested on all devices, we may want to wait for another
-    // version to make this important change.
-    if (isWebOs) {
-      await disableMediaKeys(mediaElement);
-    } else {
-      disableMediaKeys(mediaElement).catch(noop);
-    }
+    await disableMediaKeys(mediaElement);
```

Comment:
The previous code was catching errors, I guess this is why no error is thrown for the case "could not reset".
I'm not sure about where is the most appropriate place to catch errors, maybe adding a try..catch here could help?

Reply:
To me, because we're ignoring the error due to compatibility issues only (in a perfect world, failure to disable would lead to a throw). I updated the code so only compat may catch.